android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -101,7 +117,38 @@ protected void onResume() {
         realmRepo.getSessionsByLocation(location)
                 .addChangeListener((sessions, orderedCollectionChangeSet) -> {
                     mSessions.clear();
+                    String upcomingTitle = "";
+                    String track = "";
+                    String color = null;
```

Comment:
@iamareebjamal This is the error that it gives when I remove the initialisation. Please correct me if I am doing something wrong.
<img width="1247" alt="screen shot 2017-07-10 at 6 43 02 pm" src="https://user-images.githubusercontent.com/21010060/28019860-05551a42-65a0-11e7-9fd5-037cf11c0123.png">